src/components/answer/InputAnswer.jsx

```diff
@@ -87,17 +87,6 @@ class InputPropertiesResolver {
     return props;
   }
 
-  static _resolveStepRestriction(question) {
```

Comment:
@blcham I accidentally deleted this `_resolveStepRestriction` function, that is the reason the tests are not passing. My mistake

Reply:
solved in new commit